clientSrc/components/Header.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react'
-import '../styles/Header.less'
+import React from 'react';
+import '../styles/Header.less';
 
 const Header = () => {
-    return(
+    return (
         <nav className="navbar navbar-dark bg-primary">
```

Comment:
this part is not responsive so when we debug it looks like this:
http://prntscr.com/wciryl

Reply:
Huh? This change was for an ES Lint error. Are you referring to the Navbar in general?

Reply:
yes navbar, but its ok now I have submitted PR to fix it